ivy_tests/test_ivy/test_frontends/test_tensorflow/test_layers.py

```diff
@@ -0,0 +1,93 @@
+# Testing Function
+import ivy
+import numpy as np
+from hypothesis import given, strategies as st
+from hypothesis import assume
+
+# local
+import ivy_tests.test_ivy.helpers as helpers
+from ivy_tests.test_ivy.test_functional.test_core.test_statistical import (
+    statistical_dtype_values,
+)
+from ivy_tests.test_ivy.helpers import handle_cmd_line_args
+#ivy.set_backend('torch')
+#ivy.set_backend('tensorflow')
+@handle_cmd_line_args
+@given(
+    #with_unsupported_dtypes(
+    #{"1.11.0 and below": ("uint8", "bfloat16", "float16","float64"), "1.12.1": ()}, version),
+
+    dtype_and_xy=helpers.dtype_and_values(
+        available_dtypes=["float32","float32"],
+        num_arrays=2,
+        min_value=0,
+        max_value = 1,
+        shared_dtype=True,
+        min_num_dims=2,
+    ),
+
+    #dtype_and_x=helpers.dtype_and_values(
+     #   available_dtypes=helpers.get_dtypes("integer"),
+      #  num_arrays=1,
+       # min_value=0,
+        #max_value = 1,
+        #shared_dtype=True,
+        #min_num_dims=2,
+        #min_dim_size=3,
+        #shape = (2,3)
+    #),
+
+
+    #dtype_and_y=helpers.dtype_and_values(
+     #   available_dtypes=helpers.get_dtypes("float"),#"numeric"),
+      #  num_arrays=1,
+       # min_value=0,
+        #max_value = 1,
+        #shared_dtype=True,
+        #min_num_dims=2,
+        #min_dim_size=3,
+        #shape = (2,3)
+
+
+
+    #),
+    num_positional_args=helpers.num_positional_args(
+        fn_name="ivy.functional.frontends.tensorflow.top_k_categorical_accuracy"
+    ),
+    k=st.integers(min_value=1,max_value=10),
+
+
+)
+def test_tensorflow_top_k_categorical_accuracy(
+    dtype_and_xy, as_variable, num_positional_args, native_array,k
+):
+    #y_true = ivy.array(y_true, dtype=ivy.int32)
+
+    input_dtype, xy = dtype_and_xy
+    #input_dtype, y = dtype_and_y
+    #assume("float64" not in input_dtype)
+    #assume("bfloat16" not in input_dtype)
+    #assume("float16" not in input_dtype)
+    #assume("int16" not in input_dtype)
+    #assume("int64" not in input_dtype)
+    #assume("int8" not in input_dtype)
+
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        as_variable_flags=as_variable,
+        with_out=False,
+        num_positional_args=num_positional_args,
+        native_array_flags=native_array,
+        #fw=fw,
+        frontend="tensorflow",
+        fn_tree="keras.metrics.top_k_categorical_accuracy",
+        y_true=x,
+        y_pred=y,
+        k = k
+    )
+
+
+x = ivy.array([[0,0,1],[0, 1, 0]],dtype="float32")
+y = ivy.array([[0.1, 0.9, 0.8],[0.05, 0.95, 0]],dtype='float32')
```

Comment:
Once the other changes are made, try running this test.

Reply:
> Once the other changes are made, try running this test case.

passed the test on this test case. otherwise running the main function test, raises ivy.exceptions.IvyBackendException: tensorflow: argmax: 'numpy.ndarray' object has no attribute 'numpy'